src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs

```diff
@@ -51,6 +51,8 @@ public bool CanDeserialize(MediaRange mediaRange, BindingContext context)
         /// <returns>Model instance</returns>
         public object Deserialize(MediaRange mediaRange, Stream bodyStream, BindingContext context)
         {
+            bodyStream.Position = 0;

```

Comment:
Yes it's easy to check, should I submit a PR to the main repo to also check on the other deserializers ( https://github.com/NancyFx/Nancy/blob/master/src/Nancy/ModelBinding/DefaultBodyDeserializers/JsonBodyDeserializer.cs for example) ?